doc/manipulating_images/input_output.rst

```diff
@@ -104,7 +104,7 @@ datasets and atlases. They can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
-    >>> haxby_dataset = datasets.fetch_haxby(n_subjects=1)  # doctest: +SKIP
+    >>> haxby_dataset = datasets.fetch_haxby(subjects=[1])  # doctest: +SKIP
```

Comment:
How about we simply leave out the argument from this part of the documentation. It's an example that doesn't need that much detail.